app/src/main/res/values/strings.xml

```diff
@@ -336,4 +336,35 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="table">Table</string>
     <string name="how_much_columns_press_table_button_long_to_start_table">How much columns? Press table button long to start a table (insert header), press short to add a row.</string>
     <string name="example_of_a_markdown_table">Example of a Markdown table</string>
+    <string name="common_delete_lines">Delete lines</string>
+    <string name="common_attach_something">Add attachment</string>
+    <string name="common_unordered_list_hyphen">Unordered list</string>
+    <string name="common_ordered_list_number">Ordered list</string>
+    <string name="common_checkbox_list">Check list</string>
+    <string name="common_special_key">Special key</string>
```

Comment:
Please reuse strings where possible, in constrast to the `tmaid` stuff this is really about text and not about identifiers. 

"Special key" for example is already defined. In general, use here the same text for name as the value is, just with underscore_and_lower_case.

For Markdown for example you can use "Markdown Heading 1" and have markdown_heading_1. 

For all todo.txt stuff, no todo.txt in title nor value is required. (We might get HTML heading 1 in future, but for sure there is no "priority" in HTML, so no need to differntiate)

Reply:
Thank you for the review.
I'll go through it and edit all the strings.
You know I have just copied and pasted the identifiers.